generate.ts

```diff
@@ -0,0 +1,12 @@
+// Run:
+// deno run -A generate.ts
+// or
+// deno task generate
+//
+
+import { gen } from "https://etok.codes/bmt/raw/main/examples_gen.ts";
```

Comment:
Please reload the cache now that [`bmt@v0.2`](https://github.com/EthanThatOneKid/bmt/releases/tag/v0.2) is released.

Reply:
Resolved by <https://github.com/EthanThatOneKid/copyhtml/pull/1/commits/a8a8efe6435c3e06d98f4d840c1c8fd1a5f37399>.